README.md

```diff
@@ -113,3 +135,44 @@ Client Implementation
 
 `pint`, our Python-based client, is available in the Public Cloud Module, or in
 the Cloud:Tools project in OBS, as `python-susepubliccloudinfo`.
+
+Development
+-----------
+The following steps are only recommended if you want to build this project from sources, work on the codebase or test the latest development changes
+
+  1. **Install Git**
+
+         $ sudo zypper in git
+
+  2. **Install basic Ruby environment**
+
+         $ sudo zypper in ruby rubygem-bundler
+
+  3. **Install dependencies**
+
+     Install packages needed to compile Gems with native extensions:
+
+         $ sudo zypper in gcc make ruby-devel ruby2.5-rubygem-rack
+
+  4. **Clone public-cloud-info-service repository and install Gem dependencies**
+
+         $ git clone https://github.com/SUSE-Enceladus/public-cloud-info-service.git 
+         $ cd public-cloud-info-service 
+         $ bundle install
+
+  5. **Clone pint-data repository to import the xml files**
+
+         $ git clone gitlab@gitlab.suse.de:public-cloud-team/pint-data.git 
```

Comment:
Alternatively, you can use the included `spec/fixtures` data; e.g.

`FRAMEWORKS=spec/fixtures/frameworks.xml rackup`

Reply:
Since the gitlab data is private, and this project is public, it would be better to have the README reflect the public data sample.

Reply:
Yes, I completely missed that. There cannot be any references to gitlab.suse.de here. This very pull request leaks internal to SUSE data, grmbl. We should know better.